docs/styling.md

```diff
@@ -0,0 +1,241 @@
+    export default createTheme(
+      base,
+      breadcrumb,
+      minicart,
+      infoCardTheme,
+      headerTheme,
+      productQuantityTheme,
+      minicartTheme,
+      searchControlsTheme,
+      searchSuggestionsTheme,
+      searchBarTheme,
+      searchTheme,
+      custom,
+      sliderTheme,
+      offerTheme,
+      productSummaryTheme,
+      productDetailsTheme,
+      loginTheme,
+      authTheme
+    )
+    ```
+## Step by step
+
+### Overwriting CSS styles of the Store UI components
+
+Take the following steps as a guide to style Store UI components containing a pre-defined `themes.ts` style sheet.
+
+Once you [set up your SFJ project]() from our [SFJ starter](https://github.com/vtex-sites/storecomponents.store), you're ready to start styling your VTEX store.  
+
+1. Open up your SFJ project in the code editor of your choosing.
+2. Go to the `src/@vtex/gatsby-plugin-theme-ui`.
+3. Create a Typescript file with the name of the component you want to style (e.g., `searchSuggestions.ts`).
+
+>ℹ️ *Use the [SFJ Store Theme components folder](https://github.com/vtex/faststore/tree/master/packages/gatsby-theme-store/src/components) as a reference.*
+
+4. Import the `createTheme` function and the pre-defined style sheet of the component you want to style.
+    
+    ```ts
+    import { createTheme, SxStyleProp } from '@vtex/store-ui'
+    ```
+    
+>⚠️ *Ideally, all default style sheets should be defined within the Store UI library. However, you'll notice some styles are defined within the SFJ Store Theme. We are aware of this situation and already working on it.*
+
+5. Declare a `custom` object, as in the following:
+
+    ```ts
+    const custom = {
+    
+    }
+    ``` 
+
+6. Within the `custom` object, overwrite styles respecting the hierarchy of the style sheet of that component.
+
+>⚠️ *Open the default style sheet file of the component you want to style and check the hierarchy you should follow when overwriting styles.*
+
+Take the following example from the [SearchSuggestions default style sheet (`@vtex/gatsby-theme-store/src/components/SearchSuggestions/theme.ts`)](https://github.com/vtex/faststore/blob/master/packages/gatsby-theme-store/src/components/SearchSuggestions/theme.ts). By default, we have:
+    
+    ```ts
+    // "faststore/packages/gatsby-theme-store/src/components/SearchSuggestions/theme.ts"
+        products: {
+          width: 'inherit',
+    
+          button: {
+            backgroundColor: 'primary',
+          },
+    
+          title,
+    
+          list: {
+            ...list,
+            display: 'flex',
+            flexWrap: 'nowrap',
+          },
+    
+          total: {
+            paddingTop: '10px',
+            color: 'text',
+            textDecoration: 'underline',
+            textAlign: 'center',
+            cursor: 'pointer',
+            width: '100%',
+            backgroundColor: 'white',
+          },
+        },
+    ```
+
+<img src="./images/suggestions.png">
```

Comment:
Can we define a width and a height here? The images look a little big [here](https://github.com/vtex/faststore/blob/docs/styling/docs/styling.md)

Reply:
Done